src/textual/app.py

```diff
@@ -310,11 +327,13 @@ async def _on_css_change(self) -> None:
         if self.css_file is not None:
             stylesheet = Stylesheet(variables=self.get_css_variables())
             try:
-                self.log("loading", self.css_file)
+                time = perf_counter()
                 stylesheet.read(self.css_file)
-            except StylesheetError as error:
-                self.log(error)
+                elapsed = (perf_counter() - time) * 1000
+                self.log(f"loaded {self.css_file} in {elapsed:.0f}ms")
+            except Exception as error:
```

Comment:
Forgive the catch all. There are at least 2 errors that could be thrown here. At some point I will give them all a base class.